Sites/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <title>GD2 PR2</title>
+    <link rel="stylesheet" href="Styles.css">
+</head>
+
+<body>
+    <div>
+        <div class="container clearfix">
+            <h1>
+                <div class="underline">A Day in Chicago</div>
+            </h1> <br>
+
+            <div class="laptop">
+
+                <h2>Follow me through a day over my spring break while visiting my sister at The School of the Art
+                    Institute of Chicago! Impressively, I stayed awake from 10 AM to 11 PM, scroll to view my day in
+                    hourly
+                    photos! </h2>
+            </div>
+
+            <div class="laptop">
+
+                <h3>
+                    Hour 1 [ 10:16 ] </h3> <img src="img/Hour-1.jpg"
+                    alt="Picture looking up at the above-ground train and skyscrapers">
+                <figcaption> This photo is from a sunny Chicago morning on Thursday, March 1st. On the
+                    way to breakfast, we stopped to admire the city's amazing arcitecture. </figcaption>
+
+
+                <h3> Hour 2 [ 11:07 ] </h3> <img src="img/Hour-2.jpg" alt="3 girls pictured on a bridge">
+                <figcaption> Pictured from left to right is my twin sister Jenny, and my two roomates,
+                    Dasha and Damla. Together, we explored Chicago. Jenny goes to school in the city and knew
+                    all
+                    the best spots! Here they are pictured in front of the lake that was just dyed green for St.
+                    Patricks Day. <div class="a">They may look happy here, but we were all pretty hangry. They
```

Comment:
This is a pretty mysteriously named class...
Instead of using this `div`, maybe you just needed two `p` elements inside your `figcaption`?